package.json

```diff
@@ -139,19 +139,19 @@
     "enzyme": "3.10.0",
     "enzyme-adapter-react-16": "1.15.1",
     "enzyme-to-json": "3.4.3",
-    "eslint": "6.5.1",
+    "eslint": "6.6.0",
     "eslint-config-airbnb-base": "14.0.0",
-    "eslint-config-prettier": "6.4.0",
+    "eslint-config-prettier": "6.5.0",
     "eslint-formatter-pretty": "2.1.1",
     "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-jest": "22.19.0",
+    "eslint-plugin-jest": "23.0.0",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-prefer-object-spread": "1.2.1",
     "eslint-plugin-prettier": "3.1.1",
     "eslint-plugin-react": "7.16.0",
-    "eslint-plugin-react-hooks": "2.1.2",
-    "formik": "1.5.8",
-    "glob": "7.1.4",
+    "eslint-plugin-react-hooks": "2.2.0",
+    "formik": "2.0.2",
```

Comment:
I assume you quickly checked the `resetForm` calls?

Reply:
😬

Reply:
😄